sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/DataSourceV2Strategy.scala

```diff
@@ -229,7 +229,7 @@ class DataSourceV2Strategy(session: SparkSession) extends Strategy with Predicat
       throw new AnalysisException("Describing columns is not supported for v2 tables.")
 
     case DropTable(r: ResolvedTable, ifExists, purge) =>
-      DropTableExec(r.catalog, r.identifier, ifExists, purge) :: Nil
+      DropTableExec(session, r.catalog, r.identifier, ifExists, purge) :: Nil
```

Comment:
We can pass the loaded table via `r.table`

Reply:
Will do. Does it mean we also don't need to check whether the table exist in `DropTableExec`?

Reply:
Yea, it's already checked by the analyzer.

Reply:
Done.